portfolio/src/main/webapp/script.js

```diff
@@ -12,16 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-document.addEventListener('DOMContentLoaded', function() {
+document.addEventListener('DOMContentLoaded', async function() {
   // Initialise the slider (index.html) and the material box (photoGallery.html)
   elems_one = document.querySelectorAll('.slider');
```

Comment:
use const or let for all local variables, I think this one can be const, same below (sorry I didn't catch this last review)